File: prime number/code.py
t=int(input())

for i in range(t):
    count=0
    n = int(input())
    for j in range(2,n):
        if n%j == 0:
            count+=1
            break
    if count==0:
        print("prime")
    else:
        print("not prime")

File: prime number/code.js
const t = parseInt(prompt())

for (let i = 0; i < t; i++) {
    let n = parseInt(prompt())
          
    let count = 0;
    for (let div = 2; div * div <= n; div++) {
        if (n % div == 0) {
            count++;
            break;
        }

    }

    if (count == 0) {
        System.out.println("prime");
    } else {
        System.out.println("not prime");
    }
}